### PR TITLE
git-tag: add example on how to push a tag

### PR DESCRIPTION
**Pull request by ZeroOne3010**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).

I think this is a pretty crucial piece of information on how you use tags in Git.

### CODE REVIEW

**Review by spageektti — APPROVED** [review]

File: pages/common/git-tag.md [hunk 1]
```diff
@@ -28,6 +28,10 @@
 
 `git fetch --tags`
 
+- Push a tag to upstream:
```
Comment (vitorhcl) [inline pages/common/git-tag.md:31]:
```suggestion
- Push a tag to remote:
```

File: pages/common/git-tag.md [hunk 2]
```diff
@@ -28,6 +28,10 @@
 
 `git fetch --tags`
```
Comment (vitorhcl) [inline pages/common/git-tag.md:29]: I can't suggest on the line above, but

```suggestion
- Get updated tags from remote:
```

**Review by vitorhcl — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Isn't remote a better term? Upstream is also used for forks.

LGTM after this issue is addressed.

**Review by kbdharun — APPROVED** [review]
Welcome to tldr and thanks for your contribution.

LGTM, after the above suggestions.

**Comment by ZeroOne3010** [comment]
> Isn't remote a better term? Upstream is also used for forks.

Yes, good point, of course! I just copied the term from the other example without stopping to think. 🤦 I've changed both of them now!

File: pages/common/git-tag.md [hunk 3]
```diff
@@ -24,10 +24,14 @@
...
 `git tag -d {{tag_name}}`
 
-- Get updated tags from upstream:
+- Get updated tags from remote:
 
 `git fetch --tags`
 
+- Push a tag to remote:
+
+`git push origin tag {{tagname}}`
```
Comment (spageektti) [inline pages/common/git-tag.md:33]:
```suggestion
`git push origin tag {{tag_name}}`
```
Like in other examples.

**Comment by spageektti** [comment]
Thanks for your contribution @ZeroOne3010. Can you please check my suggestion so we can merge this PR?

**Pull request merged**